sample.go

```diff
@@ -0,0 +1,289 @@
+// Copyright (C) 2013-2017, The MetaCurrency Project (Eric Harris-Braun, Arthur Brock, et. al.)
+// Use of this source code is governed by GPLv3 found in the LICENSE file
+//---------------------------------------------------------------------------------------
+
+// Data integrity engine for distributed applications -- a validating monotonic
+// Samples and data used in generating a generic starter chain. See use of GenDev() in hc.
```

Comment:
I thought it would be good to split this off since it isn't really needed once you get started and load other dna, zomes, etc. It may be a candidate for splitting off so it can be included or left out external to the package.